corehq/ex-submodules/casexml/apps/case/templates/case/partials/case_hierarchy.html

```diff
@@ -37,7 +36,9 @@
 </style>
 
 <script>
-    $(function () {
+    $.when(
```

Comment:
This is the CDN issue. The script gets loaded after the following JS is executed, resulting in the console error. (Sorry, should have put this as note earlier)